src/CommitmentsV2/SFA.DAS.CommitmentsV2/Application/Queries/GetAccountSummary/GetAccountSummaryQueryHandler.cs

```diff
@@ -29,11 +29,15 @@ public GetAccountSummaryQueryHandler(Lazy<ProviderCommitmentsDbContext> dbContex
             var hasApprenticeships = await _dbContext.Value.Apprenticeships
                 .AnyAsync(a => a.Cohort.EmployerAccountId == query.AccountId, cancellationToken: cancellationToken);
 
+            var account = await _dbContext.Value.Accounts
+               .FirstAsync(a => a.Id == query.AccountId, cancellationToken: cancellationToken);
```

Comment:
Would Single() be more appropriate here?

Reply:
I don't think the other parts of this query are actually used, so in theory we could gut this handler and response and limit it only to what we want for this story.

Reply:
done